client/templates/hangout/hangout-item.html

```diff
@@ -17,7 +17,9 @@
       </label>
       <ul class="status">
         <li><i class="fa {{getType type}}"></i></li>
-        <li><label>{{getDate this}}</label></li>
+        <li><label>{{getDate this}} |  <a class="clone-hangout">
```

Comment:
Logged out users should not be able to either **see** this copy link or we should do the same check we do on a _create new hangout_ to ensure a user is logged in before they can copy a hangout.  Essentially they are creating a new hangout. 

The **check** functionality kicks off a modal asking the user to log in.  See `home.js` starting on `line 12`.